axelrod/tests/strategies/test_axelrod_second.py

```diff
@@ -158,3 +159,146 @@ def test_strategy(self):
         actions = [(D, C), (C, C), (C, D), (C, D), (D, D)]
         self.versus_test(opponent, expected_actions=actions,
                          attrs={'patsy': False})
+
+
+class TestTranquilizer(TestPlayer):
+
+    name = "Tranquilizer"
+    player = axelrod.Tranquilizer
+    expected_classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': {"game"},
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+
+    # test for initalised variables
+
+    def test_init(self):
+
+        player = axelrod.Tranquilizer()
+
+        self.assertEqual(player.num_turns_after_good_defection, 0)
+        self.assertEqual(player.opponent_consecutive_defections, 0)
+        self.assertEqual(player.one_turn_after_good_defection_ratio, 5)
+        self.assertEqual(player.two_turns_after_good_defection_ratio, 0)
+        self.assertEqual(player.one_turn_after_good_defection_ratio_count, 1)
+        self.assertEqual(player.two_turns_after_good_defection_ratio_count, 1)
+
+    def test_strategy(self):
+
+        opponent = axelrod.Bully()
+        actions = [(C, D), (D, D), (D, C), (C, C), (C, D), (D, D), (D, C), (C, C)]
+        self.versus_test(opponent, expected_actions=actions)
+
+        # Tests whether TitForTat is played given score is below 1.75
+
+        opponent = axelrod.Defector()
+        actions = [(C, D)] + [(D, D)] * 20
+        self.versus_test(opponent, expected_actions=actions, attrs={"num_turns_after_good_defection": 0,  
```

Comment:
We need to fix the PEP8 on all of these. Two options, I have a slight preference for option 2:

Option 1:

```python

`       opponent = axelrod.Defector()                                                                                                               
        actions = [(C, D)] + [(D, D)] * 20                                                                                                          
        self.versus_test(opponent, expected_actions=actions,                                                                                        
                           attrs={"num_turns_after_good_defection": 0,                                                                                
                                  "one_turn_after_good_defection_ratio": 5,                                                                           
                                  "two_turns_after_good_defection_ratio": 0,                                                                          
                                  "one_turn_after_good_defection_ratio_count": 1,                                                                     
                                  "two_turns_after_good_defection_ratio_count": 1})
```    

Option 2:


```python

`       opponent = axelrod.Defector()                                                                                                               
        actions = [(C, D)] + [(D, D)] * 20      
        expected_attrs = {"num_turns_after_good_defection": 0,                                                                                
                          "one_turn_after_good_defection_ratio": 5,                                                                           
                          "two_turns_after_good_defection_ratio": 0,                                                                          
                          "one_turn_after_good_defection_ratio_count": 1,                                                                     
                          "two_turns_after_good_defection_ratio_count": 1})                                                                                                    
        self.versus_test(opponent, expected_actions=actions, attrs=expected_attrs)                                                                                    
                           
```